digitalforms-api/src/test/java/ca/bc/gov/open/digitalformsapi/viirp/controller/DFPayloadApiTests.java

```diff
@@ -152,13 +154,16 @@ public void testDFPayloadApiDELETESuccess() throws Exception {
 
 		String correlationId = DigitalFormsConstants.UNIT_TEST_CORRELATION_ID;
 		String noticeNo = DigitalFormsConstants.UNIT_TEST_NOTICE_NUMBER;
+		
+		// Mock underlying DELETE DF Payload response in good case 
+		responseFromOrds.setStatusMessage(DigitalFormsConstants.DIGITALFORMS_SUCCESS_MSG);
+        when(dfPayloadService.deleteDFPayload(any(), any())).thenReturn(responseFromOrds);
 
 		// Create successful DELETE DF Payload and validate response
 		ResponseEntity<PostDFPayloadServiceResponse> controllerResponse = controller
 				.dfpayloadsNoticeNoCorrelationIdDelete(noticeNo, correlationId);
 		PostDFPayloadServiceResponse result = controllerResponse.getBody();
 		Assertions.assertEquals(DigitalFormsConstants.DIGITALFORMS_SUCCESS_MSG, result.getStatusMessage());
-
 	}
 
 	// TODO - need further testing for all possible PUT ORDS response codes and
```

Comment:
Again, we can add some additional tests later once we have the ORDS working. Not to worry now.